kolibri_exercise_perseus_plugin/assets/src/translator.js

```diff
@@ -3,30 +3,28 @@
 import { createTranslator } from 'kolibri.utils.i18n';
```

Comment:
In latest 0.7.x, I added an eslint disable to the top of this file. I forget why but it was necessary to enable building on 0.8.x. Could you rerun this using that?

Reply:
It's already on line 1:

![image](https://user-images.githubusercontent.com/7193975/37112331-ee1ecfc6-21f6-11e8-941b-154fdcaa0b7c.png)

Reply:
Or am I misunderstanding?